osu.Game/Overlays/Comments/CancellableCommentEditor.cs

```diff
@@ -32,45 +27,22 @@ private void load()
             });
         }
 
-        private partial class CancelButton : OsuHoverContainer
+        private sealed partial class CancelButton : RoundedButton
         {
-            protected override IEnumerable<Drawable> EffectTargets => new[] { background };
-
-            private readonly Box background;
-
             public CancelButton()
-                : base(HoverSampleSet.Button)
             {
-                AutoSizeAxes = Axes.Both;
-                Child = new CircularContainer
-                {
-                    Masking = true,
-                    Height = 25,
-                    AutoSizeAxes = Axes.X,
-                    Children = new Drawable[]
-                    {
-                        background = new Box
-                        {
-                            RelativeSizeAxes = Axes.Both
-                        },
-                        new OsuSpriteText
-                        {
-                            Anchor = Anchor.Centre,
-                            Origin = Anchor.Centre,
-                            Font = OsuFont.GetFont(size: 12, weight: FontWeight.Bold),
-                            Margin = new MarginPadding { Horizontal = 20 },
-                            Text = CommonStrings.ButtonsCancel
-                        }
-                    }
-                };
+                Height = 25;
+                AutoSizeAxes = Axes.X;
             }
 
-            [BackgroundDependencyLoader]
-            private void load(OverlayColourProvider colourProvider)
+            protected override SpriteText CreateText() => new OsuSpriteText
             {
-                IdleColour = colourProvider.Light4;
-                HoverColour = colourProvider.Light3;
-            }
+                Anchor = Anchor.Centre,
+                Origin = Anchor.Centre,
+                Font = OsuFont.GetFont(size: 12, weight: FontWeight.Bold),
+                Margin = new MarginPadding { Horizontal = 20 },
+                Text = CommonStrings.ButtonsCancel
+            };
```

Comment:
This can probably be removed completely (replaced with a `Text = CommonStrings.ButtonsCancel`).